qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -712,7 +712,12 @@ def convert_parametric(self, instruction):
         """
         t0 = instruction.t0
         channel = self.get_channel(instruction.ch)
-        pulse = ParametricPulseShapes[instruction.pulse_shape].value(**instruction.parameters)
+        parameters = instruction.parameters
```

Comment:
I'm not sure how this was working previously?

Reply:
Apparently this conversion was happening in the provider. It seems like this update will be required for the dumping of files. Its possible I could do this conversion in the mock backend loader @mtreinish ?

Reply:
I'm not sure I fully follow how this gets used. But yeah we can probably port whatever we need into the fake provider class. We'll probably add a method to the base class to extract the json loading bit and PulseDefaults object creation for all the pulse backends. Right now they just all have lines like:

https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/test/mock/backends/armonk/fake_armonk.py#L55-L63

to do this. If we need to add more common code we should do that in a single place instead of multiple